src/coreclr/jit/flowgraph.cpp

```diff
@@ -3,6 +3,12 @@
 
 #include "jitpch.h"
 
+#ifdef TARGET_WASM
+#include "llvm.h"
+#define max(a, b) (((a) > (b)) ? (a) : (b))
+#define min(a, b) (((a) < (b)) ? (a) : (b))
+#endif // TARGET_WASM
```

Comment:
We should figure out a `CodeGenInterface`-like solution at one point to avoid including all the LLVM headers into other compiler files.